templates/nomad.yaml

```diff
@@ -0,0 +1,37 @@
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    app: "nomad"
+  name: "nomad"
+spec:
+  ports:
+  - port: 8000
+    targetPort: 8000
+  selector:
+    app: "nomad"
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: "nomad"
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      app: "nomad"  
+  replicas: 1
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: "nomad"
+    spec:
+      containers:
+      - image: "nomad_dev"
+        command: ["poetry","run", "uvicorn", "nomad.main:app", "--reload", "--host=0.0.0.0"]
+        imagePullPolicy: IfNotPresent
+        name: "nomad"
+        ports:
+        - containerPort: 8000
```

Comment:
How might you move this value to `values.yaml`?

Reply:
Frankly, there are a lot of things which will go into the values.yaml moving forward.
like getting the 
`image-hash, image-repo` to update the deployment.
I wanted to properly concentrate on the `values.yaml`  when  i get to the `nomad_builder` stage.

For now the focus was to get the helm chart running so that i can go to the next step
This is more like a place-holder but will refine the helm chart going forward 

However, I agree with the point of not having  un-necessary value in the master branch